back/engines/commercial/id_vienna_saml/id_vienna_saml.gemspec

```diff
@@ -0,0 +1,20 @@
+$LOAD_PATH.push File.expand_path('lib', __dir__)
+
+# Maintain your gem's version:
+require 'id_vienna_saml/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = 'id_vienna_saml'
+  s.version     = IdViennaSaml::VERSION
+  s.summary     = "Authentication method using Vienna's Standardportal (SAML)"
+  s.authors     = ['CitizenLab']
+  s.licenses    = ['CitizenLab Commercial License V1']
+  s.files = Dir['{app,config,db,lib}/**/*', 'Rakefile', 'README.md']
+
+  s.add_dependency 'omniauth-saml', '~> 1.10.3'
```

Comment:
Can't use the most recent gem version because that requires bumping Omniauth to version 2.0, including all the existing omniauth gems we already use. I've explored that option for half a day, but it's a lot of work!